enterprise/frontend/src/metabase-enterprise/audit_app/components/AuditParameters.jsx

```diff
@@ -52,22 +53,21 @@ export default class AuditParameters extends React.Component {
     return (
       <div>
         <div className="pt4">
-          {parameters &&
-            parameters.map(({ key, placeholder }) => (
-              <input
-                className="input mr2"
-                key={key}
-                type="text"
-                value={inputValues[key] || ""}
-                placeholder={placeholder}
-                onChange={e => {
-                  this.changeValue(key, e.target.value);
-                }}
-              />
-            ))}
+          {parameters.map(({ key, placeholder, icon }) => (
+            <AuditParametersInput
+              key={key}
+              type="text"
+              value={inputValues[key] || ""}
+              placeholder={placeholder}
+              onChange={value => {
+                this.changeValue(key, value);
+              }}
+              icon={icon}
+            />
+          ))}
           {buttons &&
```

Comment:
Do we need `buttons &&` here?
Above we remove `parameters &&` from line 55.

Reply:
Buttons are optional, most of the views does not have them